File: poh20/cpp/7.cpp
//usr/bin/env true; tmpfile=$(mktemp); g++ -O2 -std=gnu++11 -xc++ -o $tmpfile $0 && $tmpfile "$@"; rm $tmpfile; exit
#import<iostream>
#import<string>
#import<algorithm>
using namespace std;
int main(){
int n;string s,t;cin>>n>>s>>t;
sort(s.begin(),s.end());
sort(t.begin(),t.end());
cout<<(s==t?"Yes":"No")<<endl;}

File: poh20/rust/7.rs
//usr/bin/env true; tmpfile=$(mktemp); rustc -o $tmpfile $0 && $tmpfile "$@"; rm $tmpfile; exit
fn l()->String{let mut x=String::new();std::io::stdin().read_line(&mut x).ok();return x;}
fn main(){
l();
let mut a:Vec<char>=l().trim().chars().collect();a.sort();
let mut b:Vec<char>=l().trim().chars().collect();b.sort();
println!("{}",if a==b{"Yes"}else{"No"});
}
